frontend/src/redux/filterSlice.test.ts

```diff
@@ -19,17 +18,10 @@ describe('filterSlice', () => {
     expect(state.namespaces).toEqual(new Set(namespaces));
   });
 
-  it('should handle setSearchFilter', () => {
```

Comment:
since the table itself has a search feature we don't really need the search filter in the header and in this slice, after discussion with @joaquimrocha we decided to just leave the namespace filter there